extensions/blocks/revue/view.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { each } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+import { __ } from '@wordpress/i18n';
```

Comment:
Instead of pulling `__` here, maybe we could set each string in PHP, and then make each string visible when necessary, kind of like it's done for the Mailchimp block?

Reply:
Fixed in b49c0bf68dcf4689410448e47390fb48bd43badf

(And damn me, it was the obvious thing to do 🤦‍♂ )